envs/example/allinone-centos/group_vars/all.yml

```diff
@@ -3,7 +3,7 @@ stack_env: example_allinone
 floating_ip: 172.16.0.100
 xtradb:
   sst_auth_password: asdf
-
+validate_certs: no
```

Comment:
nit: I know this is perfectly legal (probably more "correct" than True/False in ansible world), but would be nice to make all of these booleans consistent.